testbed/tests/widgets/test_textinput.py

```diff
@@ -94,7 +94,7 @@ async def test_on_change_user(widget, probe, on_change):
 
     for count, char in enumerate("Hello world", start=1):
         await probe.type_character(char)
-        await probe.redraw(f"Typed {char!r}")
+        await probe.redraw(f"Typed {char!r}", delay=0.02)
```

Comment:
This should have a comment explaining what the delay is for, and what would happen without it.